modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractScalaCodegen.java

```diff
@@ -27,6 +27,7 @@
     protected String modelPropertyNaming = "camelCase";
     protected String invokerPackage = "io.swagger.client";
     protected String sourceFolder = "src/main/scala";
+    protected boolean stripPackageName = true;
```

Comment:
this option should be false by default to maintain backward compatibility. Not everyone is deploying to Kubernetes.

Reply:
`stripPackageName = true` *is* retaining backward compatibility. I'm just trying to make it configurable so it can be set to `false`

Reply:
oh, sorry I misunderstood it.